<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserHubWindow</class>
 <widget class="QMainWindow" name="UserHubWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Animeplex - User Hub Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="StartWindow_BottomFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border-image: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="UserHubWindow_BackBtnFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QPushButton" name="UserHubWindow_LogoutBtn">
          <property name="minimumSize">
           <size>
            <width>95</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>95</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	color: white;	
	border: none;
}

QPushButton::hover {
	background-color: rgb(186, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>LOGOUT</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="UserHubWindow_EmailFrame">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>601</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	padding-bottom: 3px;
	padding-right: 5px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="UserHubWindow_EmailLabel">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: white;
}</string>
          </property>
          <property name="text">
           <string>PLACE_EMAIL_HERE</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="UserHubWindow_WindowBtnsFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>801</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="UserHubWindow_ViewAllAnimeBtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>175</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(18, 18, 18);
	color: white;
	border: none;
}

QPushButton::hover {
	background-color: rgb(140, 4, 24);
}</string>
          </property>
          <property name="text">
           <string>All Anime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="UserHubWindow_ViewWatchedAnimeBtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>175</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(18, 18, 18);
	color: white;
	border: none;
}

QPushButton::hover {
	background-color: rgb(140, 4, 24);
}</string>
          </property>
          <property name="text">
           <string>Watched Anime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="UserHubWindow_ViewFavoritedAnimeBtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>175</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(18, 18, 18);
	color: white;
	border: none;
}

QPushButton::hover {
	background-color: rgb(140, 4, 24);
}</string>
          </property>
          <property name="text">
           <string>Favorited Anime</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
